core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -552,6 +553,7 @@ public static synchronized void resetAllMetrics() {
       METRIC_REGISTRY.remove(timerName);
       METRIC_REGISTRY.timer(timerName);
     }
+    LAST_REPORTED_METRICS.clear();
```

Comment:
Was this a bug? Is there a test case for this?

Reply:
Yeah, it's a bug, we clears the metrics stored, but forget to clear the last_reported_metrics which may report negative value to the leading master (incorrectness).

Reply:
Added a MetricsSystemTest.testResetAllMetrics() test. If this line is not added, the test will fail.